plugins/modules/azure_rm_registrationassignment.py

```diff
@@ -0,0 +1,381 @@
+description:
+    - Create, update and delete instance of Azure RegistrationAssignment.
+options:
+    scope:
+        description:
+            - Scope of the resource.
+        required: true
+        type: str
+    registration_assignment_id:
+        description:
+            - Guid of the registration assignment.
+        type: str
+    expand_registration_definition:
+        description:
+            - Tells whether to return registration definition details also along with registration assignment details.
+        type: bool
+    properties:
+        description:
+            - Properties of a registration assignment.
+        type: dict
+        suboptions:
+            registration_definition_id:
+                description:
+                  - Fully qualified path of the registration definition.
+                required: true
+                type: str
+            registration_definition:
+                description:
+                    - Registration definition inside registration assignment.
+                type: dict
+                suboptions:
+                    properties:
+                        description:
+                            - Properties of registration definition inside registration assignment.
+                        type: dict
+                    plan:
+                        description:
+                            - Plan details for the managed services.
+                        type: dict
+                        suboptions:
+                            name:
+                                description:
+                                    - The plan name.
+                                required: true
+                                type: str
+                            publisher:
+                                description:
+                                    - The publisher ID.
+                                required: true
+                                type: str
+                            product:
+                                description:
+                                    - The product code.
+                                required: true
+                                type: str
+                            version:
+                                description:
+                                    - The plan's version.
+                                required: true
+                                type: str
+    state:
+        description:
+            - Assert the state of the RegistrationAssignment.
+            - Use C(present) to create or update an RegistrationAssignment and C(absent) to delete it.
+        default: present
+        type: str
+        choices:
+            - absent
+            - present
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
+author:
+    - GuopengLin (@t-glin)
+    - Fred-Sun (@Fred-Sun)
+    - Haiyuan Zhang (@haiyuazhang)
+
+'''
+
+EXAMPLES = '''
```

Comment:
could we add another example to show that scope can be in the subscription/group level?